src/terminal/parser/InputStateMachineEngine.cpp

```diff
@@ -480,6 +496,13 @@ bool InputStateMachineEngine::ActionCsiDispatch(const wchar_t wch,
             success = _pDispatch->WindowManipulation(static_cast<DispatchTypes::WindowManipulationType>(function),
                                                      remainingArgs);
             break;
+        case CsiActionCodes::Win32KeyboardInput:
+        {
+            // Use WriteCtrlKey here, because that will take extra steps to make

```

Comment:
Sorry, you mean "Use WriteCtrlKey here even for things that are not necessarily control keys", right?